### ISSUE TITLE
version 5.1.0 compile fail

### ISSUE THREAD

**Issue by MagicalWater** [marcglasberg/back_button_interceptor#18]
console message 
```
../../development/flutter/.pub-cache/hosted/pub.dartlang.org/back_button_interceptor-5.1.0/lib/src/back_button_interceptor.dart:30:31: Error: Property 'handlePopRoute' cannot be accessed on 'WidgetsBinding?' because it is potentially null.
     - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('../../development/flutter/packages/flutter/lib/src/widgets/binding.dart').
    Try accessing using ?. instead.
          WidgetsBinding.instance.handlePopRoute;
                                  ^^^^^^^^^^^^^^
```

WidgetsBinding.instance is null safety

**Comment by noga-dev** [comment]
run ```flutter upgrade```

**Comment by FiShelly** [comment]
may be use 6.x is better? some project do not want to update flutter version.
@noga-dev

**Comment by noga-dev** [comment]
There's no reason not to upgrade. It's stable.

**Comment by marcglasberg** [comment]
@FiShelly Yes, it would have been better. I made a mistake publishing it as 5.1.0.

Just fix the version by removing the `^` in your pubspec.yaml file.

```
  back_button_interceptor: 5.1.0
```

**Comment by marcglasberg** [comment]
@FiShelly I retracted version 5.1.0 and published another one as 6.0.0.

Try running `pub get` again, or else just fix the version to 5.0.2 and remove the ^ in your pubspec.yaml file.

```
  back_button_interceptor: 5.0.2
```

**Comment by FiShelly** [comment]
> @FiShelly I retracted version 5.1.0 and published another one as 6.0.0.
> 
> Try running `pub get` again, or else just fix the version to 5.0.2 and remove the ^ in your pubspec.yaml file.
> 
> ```
>   back_button_interceptor: 5.0.2
> ```

all right, thx~
